.claude/skills/legal-petition/assets/petition-style.css

```
@@ -0,0 +1,153 @@
+ * Legal Petition CSS - Brazilian Court Format
+ *
+ * Paper: A4 (210mm x 297mm)
+ * Font: Times New Roman, 12pt
+ * Line spacing: 1.5
+ * Margins: Top/Right/Bottom 2.5cm, Left 3cm
+ * Text: Justified
+ * Page numbers: Bottom center
+ */
+
+@page {
+    size: A4;
```

Stylelint (17.3.0)

[error] Expected "A4" to be "a4" (value-keyword-case)

(value-keyword-case)